hypha/apply/projects/templates/application_projects/project_approval_detail.html

```diff
@@ -156,7 +157,7 @@ <h5>{% trans "Actions to take" %}</h5>
                         <a class="button button--bottom-space button--primary button--full-width {% if user_can_approve %} is-disabled {% endif %}" href="{% url 'apply:projects:edit' pk=object.pk %}">{% trans "Edit PAF" %}</a>
                     {% endif %}
                     <div class="dropdown">
-                      <button class="button button--white dropbtn button--bottom-space button--full-width" type="button" data-dropdown-target="#dropdown-content-download">
+                      <button class="button button--white dropbtn button--bottom-space button--full-width" style="padding:0.5em 53px" type="button" data-dropdown-target="#dropdown-content-download">
```

Comment:
Let's not use custom spacing unless it's absolutely needed. Use `tailwind-css` classes for the spacing. 

Here you can use `py-1 py-3`, also the current default padding on the buttons is too much, i've modified it here in my PR, you can also change the default padding: 

https://github.com/HyphaApp/hypha/pull/3149/files#diff-6ae81798f0bc08223ae8def2462a5eed782ace1b07ef3d4da63a7e21ee8be5e8R81

![image](https://user-images.githubusercontent.com/236356/232687064-da98dee9-e721-440d-822e-6e5eec7b47b4.png)


hypha/static_src/src/sass/apply/abstracts/_mixins.scss

Reply:
I was thinking to convert them into a css class after testing but I think it might be better if we can use tailwind classes directly. I'll try that out.